src/test/java/browsertests/day4.java

```diff
@@ -0,0 +1,21 @@
+package browsertests;
+
+import org.testng.annotations.Test;
+
+public class day4 {
```

Comment:
Classes should start with capital letters and not include numbers.

Consider naming this to `DayFour`